python_scripts/01_tabular_data_exploration.py

```diff
@@ -170,6 +170,16 @@
 # %%
 adult_census["sex"].value_counts()
 
+# %% [markdown]
+# Note that there is an important imbalance on the data collection concerning
+# the number of male/female samples. Be aware that any kind of data imbalance
+# will impact the generalizability of a model trained on it. Moreover, it can
+# lead to
+# [fairness](https://docs.microsoft.com/en-us/azure/machine-learning/concept-fairness-ml#what-is-machine-learning-fairness)
+# problems if used naively when deploying a real life setting. Later in this
+# course we will see how to decompose performance metrics to take into account
+# such data imbalances.
```

Comment:
You are right, what we do on M7 is setting a baseline with `DummyClassifier(strategy="most_frequent")` to avoid overoptimistic interpretations.
Do you think it is worth mentioning we do this?

Reply:
We can later expand M7 to decompose the error metrics per-gender subgroup but let's do that in another PR.